wikihowunofficialapi/__init__.py

```diff
@@ -289,6 +323,59 @@ def _parse_views(self, soup):
             self._views = int(''.join(
                 (span[span.find('>')+1: span.find('</span>')]).split(',')))
 
+    def _parse_co_authors(self, soup):
+        co_authors_html = soup.find('div', {'class': 'sp_box sp_stats_box'})
+        if not co_authors_html:
+            raise ParseError
+        else:
+            div = co_authors_html.findAll('div', {'class': 'sp_text'})
+            span = str(div[0].find('span', {'class': 'sp_text_data'}))
+            self._co_authors = int(''.join(
+                (span[span.find('>')+1: span.find('</span>')]).split(',')))
+
+    def _parse_references(self, soup):
+        references_html = soup.findAll('a', {'class': 'external free'})
+        count = 0
+        if not references_html:
+            return None
+        else:
+            for reference in references_html:
+                count += 1
+        self._references = count
+
+    def _parse_summary(self, soup):
+        summary_html = soup.find('p', {'id': 'summary_text'})
+        if not summary_html:
+            return None
+        else:
+            self._summary = summary_html.text
```

Comment:
At the end of the summary it prints "Did this summary help you?YesNo", which should not be included in it. Could you see what can be done?

Reply:
Okay, I will look into it!